larq/models.py

```diff
@@ -81,6 +81,27 @@ def _number_as_readable_str(num: float) -> str:
     return num + unit
 
 
+def _memory_as_readable_str(num_bits: int) -> str:
+    """Generate a human-readable string for the memory size.
+
+    1 KiB = 1024 B; we use the binary prefix (KiB) [1] instead of the decimal prefix
+    (KB) to avoid any confusion with multiplying by 1000 instead of 1024.
+
+    [1] https://en.wikipedia.org/wiki/Binary_prefix
```

Comment:
Please could you replace this with a non-wikipedia reference?

Reply:
Done in e31028f.